media/css/firefox/firstrun/firstrun-quantum.scss

```diff
@@ -312,325 +312,44 @@ html[dir="rtl"] #main-header {
     }
 }
 
-// More complex interactions for other platforms
-html.osx,
-html.win8up {
+#main-header h1,
+#main-header .content,
+#main-header a {
+    opacity: 0;
+    transform: translateY(-5px);
+}
 
-    #main-header h1,
-    #main-header .content,
-    #main-header a {
-        opacity: 0;
-        transform: translateY(-5px);
-    }
+#fxa-iframe-config {
+    opacity: 0;
+    transform: scale(.8);
+}
 
+#scene[data-content=true] {
     #fxa-iframe-config {
-        opacity: 0;
-        transform: scale(.8);
-    }
-
-    #scene[data-animate=true] {
-        .wave1 {
-            animation: Wave1 17.26s infinite forwards;
-        }
-
-        .wave2 {
-            animation: Wave2 11.08s infinite forwards;
-        }
-
-        .wave3 {
-            animation: Wave3 7.51s infinite forwards;
-        }
-
-        .wave4 {
-            animation: Wave4 15.14s infinite forwards;
-        }
-
-        .wave5 {
-            animation: Wave5 13.46s infinite forwards;
-        }
-
-        .wave6 {
-            animation: Wave6 20.55s infinite forwards;
-        }
-
-        .wave7 {
-            animation: Wave7 26.55s infinite forwards;
-        }
-
-        .wave8 {
-            animation: Wave8 27.43s infinite forwards;
-        }
-    }
-
-    #scene[data-content=true] {
-        #fxa-iframe-config {
-            opacity: .85;
-            transform: scale(1);
-            transition: opacity 1s 2s, transform 1s 2s;
-        }
-
-        #main-header {
-            h1,
-            .content,
-            a {
-                opacity: 1;
-                transform: translateY(0);
-                transition: transform .5s, opacity .8s;
-            }
-
-            h1 {
-                transition-delay: 1.8s;
-            }
-
-            .content {
-                transition-delay: 2s;
-            }
-
-            a {
-                transition-delay: 2.2s;
-            }
-        }
+        opacity: .85;
```

Comment:
I see some improvements when I bump this to 1

Reply:
I don't get a huge performance boost locally but I also don't know if the slight transparency is a huge benefit either, so maybe any little bit helps?